frame/support/src/lib.rs

```diff
@@ -2003,18 +2003,18 @@ pub mod pallet_prelude {
 /// 	* storage names, hasher, prefixes, default value
 /// 	* error , error, constant,
 /// * manually check that:
-/// 	* Origin is moved inside macro unser `#[pallet::origin]` if it exists
-/// 	* ValidateUnsigned is moved inside macro under `#[pallet::validate_unsigned)]` if it exists
-/// 	* ProvideInherent is moved inside macro under `#[pallet::inherent)]` if it exists
-/// 	* on_initialize/on_finalize/on_runtime_upgrade/offchain_worker are moved to Hooks
+/// 	* `Origin` is moved inside the macro under `#[pallet::origin]` if it exists
+/// 	* `ValidateUnsigned` is moved inside the macro under `#[pallet::validate_unsigned)]` if it exists
+/// 	* `ProvideInherent` is moved inside macro under `#[pallet::inherent)]` if it exists
+/// 	* `on_initialize`/`on_finalize`/`on_runtime_upgrade`/`offchain_worker` are moved to `Hooks`
 /// 		implementation
-/// 	* storages with `config(..)` are converted to genesis_config field, and their default is
+/// 	* storages with `config(..)` are converted to `GenesisConfig` field, and their default is
 /// 		`= $expr;` if the storage have default value
-/// 	* storages with `build($expr)` or `config(..)` are built in genesis_build
-/// 	* add_extra_genesis fields are converted to genesis_config field with their correct default
-/// 		if specified
-/// 	* add_extra_genesis build is written into genesis_build
-/// * storages now use PalletInfo for module_prefix instead of the one given to decl_storage:
+/// 	* storages with `build($expr)` or `config(..)` are built in `GenesisBuild::build`
+/// 	* `add_extra_genesis` fields are converted to `GenesisConfig` field with their correct
+/// 		default if specified
+/// 	* `add_extra_genesis` build is written into `GenesisBuild::build`
+/// * storages now use `PalletInfo` for `module_prefix` instead of the one given to `decl_storage`:
 /// 	Thus any use of this pallet in `construct_runtime!` should be careful to update name in
 /// 	order not to break storage or to upgrade storage (moreover for instantiable pallet).
 /// 	If pallet is published, make sure to warn about this breaking change.
```

Comment:
```suggestion
/// 	Thus any use of this pallet in `construct_runtime!` should be careful to update the name
/// 	so that storage is neither broken nor upgraded (especially for instantiable pallets).
/// 	If the pallet is published, make sure to warn about this breaking change.
```
The meaning is slightly unclear to me here as well. Would the rephrasing capture your intention?

(Aside: Sorry for picking apart the docs around your changes, but my guess is that you want to improve the docs generally. And I didn't look at the surrounding text much on my initial review.)

Reply:
actually ppl should either:
* check that the prefix used by storages hasn't change
* or do a storage migration

I updated instruction in https://github.com/paritytech/substrate/pull/7922/commits/f6b100cd8fdd22b1358500e20caa3c140eb6b61d
I hope it makes sense